apps/cowswap-frontend/src/modules/ordersTable/containers/OrderRow/EstimatedExecutionPrice.tsx

```diff
@@ -140,6 +163,17 @@ export function EstimatedExecutionPrice(props: EstimatedExecutionPriceProps) {
     </>
   )
 
+  // Reset approveClicked state after 3 seconds
+  useEffect(() => {
```

Comment:
Added loading state after clicking to "Approve" because it takes some time to calculate "shouldZeroApprove" value